docs/user-guide/walkthrough/k8s201.md

```diff
@@ -90,10 +94,8 @@ contains the desired changes:
 
 {% include code.html language="yaml" file="deployment-update.yaml" ghlink="/docs/user-guide/walkthrough/deployment-update.yaml" %}
 
-Download ./deployment-update.yaml and copy to your local directory.
-
 ```shell
-kubectl apply -f ./deployment-update.yaml
+kubectl apply -f https://k8s.io/docs/user-guide/walkthrough//deployment-update.yaml

```

Comment:
Please use this path instead:
`/docs/user-guide/walkthrough//deployment-update.yaml`